core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -238,4 +249,63 @@ protected static String fullTableName(String catalogName, TableIdentifier identi
 
     return sb.toString();
   }
+
+  @Override
+  public Table createTable(
```

Comment:
Why is this necessary? Won't this be handled in `BaseMetastoreCatalogTableBuilder.create`?

Reply:
You are right, this is not needed. Removing it.